ocs_ci/utility/aws.py

```diff
@@ -1533,14 +1540,18 @@ def get_hosted_zone_id(self, cluster_name):
         cluster_name = cluster_name or config.ENV_DATA["cluster_name"]
         base_domain = config.ENV_DATA["base_domain"]
         hosted_zone_name = f"{cluster_name}.{base_domain}."
+        logger.info(hosted_zone_name)
+        logger.info(base_domain)
+        logger.info(cluster_name)
```

Comment:
Same as above, please add some description to the log message (and maybe consider if they are worth `info` level or `debug` will be sufficient).